.github/workflows/nightly-ci.yml

```
@@ -326,6 +340,26 @@ jobs:
+    name: fastvideo-runtime # This name overlaps with other fastvideo jobs to group them in the UI
+    needs: [fastvideo-build, resolve-source-sha, compute-release-mode]
+    if: ${{ needs.compute-release-mode.outputs.run_tests == 'true' }}
+    uses: ./.github/workflows/shared-test.yml
+    with:
+      test_suite_name: fastvideo
+      test_type: Test
+      amd_runner: prod-tester-amd-gpu-v2 # This runner is overridden for ARM platform
+      target_tag_plain: ${{ needs.fastvideo-build.outputs.target_tag_plain }}
+      cuda_version: '["13.1"]'
+      platform: '["amd64", "arm64"]' # arm64 for CPU tests, single GPU tests are skipped
+      enable_coverage: true
```

actionlint (1.7.12)

[error] input "enable_coverage" is typed as string by reusable workflow "./.github/workflows/shared-test.yml". bool value cannot be assigned

(expression)